pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -393,17 +394,20 @@ private void ensureMinReplicas(TableConfig config) {
   public String rebalance(
       @ApiParam(value = "Name of the table to rebalance") @Nonnull @PathParam("tableName") String tableName,
       @ApiParam(value = "offline|realtime") @Nonnull @QueryParam("type") String tableType,
-      @ApiParam(value = "true|false") @Nonnull @QueryParam("dryrun") Boolean dryRun,
-      @ApiParam(value = "true|false") @DefaultValue("false") @QueryParam("includeConsuming") Boolean includeConsuming) {
+      @ApiParam(value = "true|false") @Nonnull @DefaultValue("true") @QueryParam("dryrun") Boolean dryRun,
+      @ApiParam(value = "true|false") @DefaultValue("false") @QueryParam("includeConsuming") Boolean includeConsuming,
+      @ApiParam(value = "True if you are trying to bootstrap replica groups for a table. "
+          + "False if you are rebalancing the replica group table or other typed table")
+        @DefaultValue("false") @QueryParam("replicaGroupBootstrap") Boolean bootstrap) {
```

Comment:
i still dont see a need for this flag. please remove this and provide simpler atomic primitives. We can always add the more complex funcitonality later, but once we provide an API we are stuck with it forever,

Reply:
ok i removed this